contracts/child/ChildERC20.sol

```diff
@@ -49,6 +49,18 @@ contract ChildERC20 is ChildToken, ERC20, LibTokenTransferOrder, ERC20Detailed {
     parent = _parent;
   }
 
+  function allowance(address, address) public view returns (uint256) {
+    revert("disabled feature");
+  }
+
+  function approve(address, uint256) public returns (bool) {
+    revert("disabled feature");
+  }
+
+  function transferFrom(address from, address to, uint256 value) public returns (bool){
+    revert("disabled feature");
+  }
```

Comment:
because childToken doesn't inherit ERC20.

Reply:
let me check again with https://www.python.org/download/releases/2.3/mro/
and anyway these methods shouldn't be in childToken since we have different methods disabled for erc20/erc721. both share a single common method.
with new refactoring, these disabled methods are in ERC20Mixin.